core/templates/dev/head/pages/exploration_editor/simple_editor_tab/ScrollSyncDirective.js

```diff
@@ -13,33 +13,30 @@
 // limitations under the License.
 
 /**
- * @fileoverview Directive used to mark parts of the editor that have a sidebar
+ * @fileoverview Directives used to mark parts of the editor that have a sidebar
  * item associated to it, see ScrollSyncService.js.
  */
 
-oppia.directive('scrollSync', ['ScrollSyncService', 'QuestionIdService',
-  function(ScrollSyncService, QuestionIdService) {
+oppia.directive('scrollSync', ['ScrollSyncService',
```

Comment:
Could we get rid of scrollSync altogether, and move the addTarget() call to the anchorHeader controller?